src/responder/nss/nss_cmd.c

```diff
@@ -826,6 +826,11 @@ static errno_t sss_nss_setnetgrent(struct cli_ctx *cli_ctx,
         goto done;
     }
 
+    cmd_ctx->enum_limit = 1;
```

Comment:
I understand that this patch makes `enum_limit` obsolete for netgroups code.
But to read `enum_limit = 1` is somewhat confusing.

Reply:
Would a comment here make sense?